dist/index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/84967187a9.js" crossorigin="anonymous"></script>
+    <title>TO DO LIST</title>
+<script defer src="index.js"></script></head>
+
+<body>
+    <div class="container">
+        <ul class="item_form">
+            <li class="header">
+                <h1>Today's Todo</h1>
+                <i class="fas fa-sync-alt"></i>
+            </li>
+            <li class="inputContainer">
+                <form id="sub_form">
+                    <input type="text" id="new-item" placeholder="Add to your list...">
+                    <div class="submit-item">
+                        <button id="submit_item" type="submit"><i class="fa fa-arrow-left"></i></button>
+                    </div>
+                </form>
```

Comment:
:black_square_button: I can see you implemented the `form` and the `div` tags inside the `li` element, but for the HTML semantic requirements, the `form`  or `div` can not be nested inside a `li` element.